cmd/server/files.go

```diff
@@ -60,16 +60,20 @@ func getCashLetterId(w http.ResponseWriter, r *http.Request) string {
 
 func getFiles(logger log.Logger, repo ICLFileRepository) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		if requestID := moovhttp.GetRequestID(r); requestID != "" {
+			logger.Set("requestID", requestID)
```

Comment:
on calls to `Set` or `With`, we have to redeclare the logger: `logger = logger.Set()`

Reply:
Ohhh that's good to know. Thanks for pointing it out.